src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -979,19 +1038,17 @@ function MoneyRequestConfirmationList({
         // @ts-expect-error This component is deprecated and will not be migrated to TypeScript (context: https://expensify.slack.com/archives/C01GTK53T8Q/p1709232289899589?thread_ts=1709156803.359359&cid=C01GTK53T8Q)
         <OptionsSelector
             sections={optionSelectorSections}
-            onSelectRow={canModifyParticipants ? selectParticipant : navigateToReportOrUserDetail}
+            onSelectRow={navigateToReportOrUserDetail}
             onAddToSelection={selectParticipant}
             onConfirmSelection={confirm}
             selectedOptions={selectedOptions}
-            canSelectMultipleOptions={canModifyParticipants}
-            disableArrowKeysActions={!canModifyParticipants}
+            disableArrowKeysActions
             boldStyle
             showTitleTooltip
             shouldTextInputAppearBelowOptions
             shouldShowTextInput={false}
             shouldUseStyleForChildren={false}
-            optionHoveredStyle={canModifyParticipants ? styles.hoveredComponentBG : {}}
-            footerContent={footerContent}
+            footerContent={!isEditingSplitBill && footerContent}
```

Comment:
Is this intentional?

Previously after updating amount (for a scan request while scanning) and clicking the footer button the amount cannot be updated anymore.

With the button removed, now the amount can be changed multiple times and scanning also does not stop with updating the amount.

https://github.com/Expensify/App/assets/102477862/7b62efe4-1dd8-4624-9dc6-9e1e5554b6fa

First few seconds of the video is for this PR and remaining part on staging.

Reply:
I have no idea how this change landed in my PR 😅 definitely a mistake. Thanks for catching this!